src/index.js

```diff
@@ -176,4 +177,4 @@ class WebRTCDirect {
   }
 }
 
-module.exports = WebRTCDirect
+module.exports = withIs(WebRTCDirect, { className: 'WebRTCDirect', symbolName: '@libp2p/js-libp2p-webrtc-direct/webrtcdirect' })
```

Comment:
What is the `className` used for? Can't we just have the `symbolName` instead?

Reply:
You have to pass the `className` so you can call the method, even when uglified.

```js
const instance = new WebRTCDirect();

WebRTCDirect.isWebRTCDirect(instance); // true
```

Check [this line here](https://github.com/moxystudio/js-class-is/blob/9df979b4b95298417f7a9aa28b0eb1158101aaaf/index.js#L25).